test/framework/src/main/java/org/elasticsearch/index/engine/EngineTestCase.java

```diff
@@ -1628,7 +1628,7 @@ private static LazySoftDeletesDirectoryReaderWrapper.LazyBits lazyBits(LeafReade
         throw new IllegalStateException("Can not extract lazy bits from given index reader [" + reader + "]");
     }
 
-    static CodecService newCodecService() {
+    protected static CodecService newCodecService() {
```

Comment:
This is to simplify test set up for whatever classes might inherit EngineTestCase